nuttx-configs/px4-stm32f4discovery/include/board.h

```diff
@@ -34,8 +34,8 @@
  *
  ************************************************************************************/
 
-#ifndef __CONFIG_PX4FMU_V1_INCLUDE_BOARD_H
-#define __CONFIG_PX4FMU_V1_INCLUDE_BOARD_H
+#ifndef __ARCH_BOARD_BOARD_H
```

Comment:
Please change to __CONFIG_DISCOVERY_INCLUDE_BOARD_H, the current name is too generic.